app/components/ScorecardList/ScorecardListModals.js

```diff
@@ -10,9 +11,10 @@ class ScorecardListModals extends Component {
 
   render() {
     const { translations } = this.context;
+    const scorecardCode = <Text style={{fontWeight: 'bold'}}>{ this.props.scorecardUuid }</Text>
```

Comment:
@limkimsan please use a consistent variable name for the scorecard code term(if we use Uuid so it should be scorecardUuid anywhere in the app)